### DISCUSSION TITLE
Kiosk App Exploit

### DISCUSSION THREAD

**Discussion by ray100-sketch** [3kh0/ext-remover#800] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Works on Version 115 and below. If on 115, and need to update, use CAUB, or CAUB Flags to use this exploit.

Kiosk Apps Can unblock websites, and they work fine. Sign out of your Chromebook, the click "Apps" Click on any "Test Browser" Kiosk app. For example, Riverside Elevate. Open it, click Terms of Use. Scroll, click "Privacy Policy" Then, sign-in. Then, sign in with  the one you see has any social media page, and click it. Click sign in with google, then Privacy. Then click the waffle, and Google, or YouTube. BOOM. It's unblocked. Comment if it doesn't work, and I will try my best to fix it.

Btw if you have network blocker extension your screwed.
And if your school has active log-in extensions your screwed.

**Comment by Su4o** [comment]
Why doesn't it work over ver 115? I'm on ChromeOs 118 (i still use CAUB and Metered Flag)

**Reply by ray100-sketch** [reply]
Because filters on V 115 and above will already, if not start to use sign out filtering. (Which means still filter even though you aren't signed in. It will use that you are enrolled, and function your extensions, as if you were actually signed in.)
